styles/all/template/js/collapsiblecategories.js

```diff
@@ -0,0 +1,58 @@
+(function ($) {  // Avoid conflicts with other libraries
+
+	'use strict';
+
+	$(function () {
```

Comment:
We do not need this. This is shorthand for document ready right? We do not need to document ready since this script is loaded in the footer, so it is loaded after the DOM above already exists.